chap-ai.tex

```diff
@@ -294,8 +294,8 @@ \subsection{Função de Custo como Algoritmo de Planning}
 Podemos ver que, para cada etapa do algoritmo de planning, estamos avaliando uma
 função de custo. Isto fica mais explícito na Figura~\ref{fig:planning}. Se
 considerarmos o caso especial de horizonte 1, o resultado do algoritmo seria
-idêntico ao de apenas avaliar a função de custo, como proposto na
-Sessão~\ref{sec:ai:minimize-cost-function}.
+idêntico ao de apenas avaliar a função de custo para uma única tomada de
+decisão, como proposto na Sessão~\ref{sec:ai:minimize-cost-function}.
```

Comment:
Oh dear. Vou dar um search and replace que deve ter duzentas erradas :P

On Thu, Nov 19, 2015 at 11:49 AM Guilherme Taschetto <
notifications@github.com> wrote:

> In chap-ai.tex
> <https://github.com/vantas/tc1/pull/113#discussion_r45341836>:
>
> > @@ -294,8 +294,8 @@ \subsection{Função de Custo como Algoritmo de Planning}
> >  Podemos ver que, para cada etapa do algoritmo de planning, estamos avaliando uma
> >  função de custo. Isto fica mais explícito na Figura~\ref{fig:planning}. Se
> >  considerarmos o caso especial de horizonte 1, o resultado do algoritmo seria
> > -idêntico ao de apenas avaliar a função de custo, como proposto na
> > -Sessão~\ref{sec:ai:minimize-cost-function}.
> > +idêntico ao de apenas avaliar a função de custo para uma única tomada de
> > +decisão, como proposto na Sessão~\ref{sec:ai:minimize-cost-function}.
>
> Sessão só no cinema. Aqui é seção mesmo. :P
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/vantas/tc1/pull/113/files#r45341836>.
>